contracts/mocks/ReentrancyMock.sol

```diff
@@ -38,7 +38,9 @@ contract ReentrancyMock is ReentrancyGuard {
     }
 
     function _count() private {
-        counter += 1;
+        unchecked{
+            counter += 1;
+        }
```

Comment:
we don't care about the gas efficiency of mocks. Lets leave them as they are for the sake of readability.